json_serializable/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 4.0.0-dev
+## 4.0.0
 
+- Requires Dart 2.11 or greater.
```

Comment:
Do you want to mention why you are using this hack? I'd be confused as an end user to see that you require an SDK that doesn't exist.

Reply:
Also I think @jakemac53 may have wanted to use `2.11.99` over in `build`...